tests/e2e/configuration/storage_info.py

```diff
@@ -54,13 +54,12 @@ def test_does_default_config_match():
 
     # The default value of these is dependent on the given machine.
     machine_dependent_configurations = [
-        "memory_usage",
+        "memory_res",
         "disk_usage",
         "memory_allocated",
         "allocation_limit",
-        "vm_max_map_count",
+        "",
```

Comment:
Why has this disappeared?

Reply:
Don't know really, this one snuck into PR, will revert